src/test/java/pages/RecentlyPlayedPage.java

```diff
@@ -0,0 +1,35 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class RecentlyPlayedPage extends BasePage{
+
+    public RecentlyPlayedPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+    //elements
+    static By recentlyPlayedPage = By.cssSelector("a[href='#!/recently-played']");
+    static By AllBtn = By.cssSelector("button.btn-shuffle-all");
+    static By ClearAllBtn = By.cssSelector("button[title='Clear current queue']");
+    static By NoSongsQueuedText = By.cssSelector("div.text");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```